addons/settings/fnc_initDisplayMain.sqf

```diff
@@ -28,6 +28,12 @@ if (_file != "") then {
 
 uiNamespace setVariable [QGVAR(userconfig), compileFinal str _userconfig];
 
+private _volatile = getNumber (configFile >> QGVAR(volatile)) == 1;
```

Comment:
```suggestion
private _volatile = (getNumber (configFile >> QGVAR(volatile)) == 1) && {!hasInterface};
```
also not sure, but I feel like this is the intent

Reply:
After giving it a bit of thought I guess we might want to have it run only on dedicated servers. HCs don't fiddle with settings so this should be good.